.cirrus.yml

```diff
@@ -96,6 +103,15 @@ task:
         matrix:
           CHANNEL: "2.10.5"
           CHANNEL: "2.8.1"
+      pathified_analyze_script:
```

Comment:
All the comments above apply here. Do you really need this though? I'm having trouble imagining a scenario where an inter-package breakage only happens on a legacy version.

Reply:
Good point, removed.